### DISCUSSION TITLE
Insert mode by default when using vim plugin

### DISCUSSION THREAD

**Discussion by wrnrlr** [zed-industries/zed#14927] [Feature Requests]
Hi, Is there a way to keep Zed in insert mode when opening a new file while using the vim plugin.
I like the vim plugin but only if I explicitly wanna use it. Keep up the good work.

**Comment by 0x2CA** [comment]
You should use vim mode like a painter, and you should remove the brush from the board when not painting.You should always be in normal mode.

**Comment by aspiringLich** [comment]
If anyone finds this discussion in the future this is resolved by #25067.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
